packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -1655,10 +1663,11 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
       _stopCursorTimer(resetCharTicks: false);
       _startCursorTimer();
     }
+    _updateEditingValueInProgress = false;
   }
 
-  bool _isSelectionOnlyChange(TextEditingValue value) {
-    return value.text == _value.text && value.composing == _value.composing;
+  bool _isSelectionOnlyChanged(TextEditingValue value) {
```

Comment:
I think it should stay as "Change", as I understand it.  I think it meant like: "is selection the only change"

Reply:
Done.